adhearsion.gemspec

```diff
@@ -23,7 +23,6 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'activesupport', [">= 3.0.0", "< 5.0.0"]
   s.add_runtime_dependency 'adhearsion-loquacious', ["~> 1.9"]
-  s.add_runtime_dependency 'bundler', ["~> 1.0"]
```

Comment:
I'm not sure this is quite correct. The framework still requires bundler. Can we instead relax the restriction to 1.0 or 2.0?